packages/angular/src/generators/application/application.ts

```diff
@@ -60,6 +64,16 @@ export async function applicationGeneratorInternal(
     ensureAngularDependencies(tree);
   }
 
+  options.onBoardingStatus = await getNxCloudOnBoardingStatus(
+    tree,
+    options.nxCloudToken
+  );
+  if (options.onBoardingStatus === 'unclaimed') {
+    options.connectCloudUrl = await getNxCloudAppOnBoardingUrl(
+      options.nxCloudToken
+    );
+  }
+
```

Comment:
I don't know if we need to put these onto `options`. Should we move this into `createFiles`?